src/Codec/Binary/Bech32/Internal.hs

```diff
@@ -253,6 +258,21 @@ data HumanReadablePartError
     | HumanReadablePartContainsInvalidChars [CharPosition]
     deriving (Eq, Show)
 
+-- | Like `humanReadablePartFromText`, but throws a runtime error if given
+--   invalid input.
+--
+-- It is recommended to make use of this function only when using literals.
+--
+-- For example:
+--
+-- >>> unsafeHumanReadablePartFromText "addr"
+-- HumanReadablePart "addr"
+--
+unsafeHumanReadablePartFromText :: Text -> HumanReadablePart
```

Comment:
Why not just return `Either` instead of throwing an exception from pure code?

Reply:
@erikd wrote:
> Why not just return `Either` instead of throwing an exception from pure code?

That's a good point.

The original function `humanReadablePartFromText` actually does return `Either`.

This `unsafe` variant is provided merely as a _convenience_, meant for use in code that defines literal constants (such as test code, or code that defines a human-readable prefix that is fixed for all eternity).

In our `cardano-wallet` local "fork" of the `bech32` library, we define a similar function:

https://github.com/input-output-hk/cardano-wallet/blob/master/lib/bech32/src/Codec/Binary/Bech32/Internal.hs#L268

It's used in the following places:
```hs
lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs
143:        let hrp = Bech32.unsafeHumanReadablePartFromText "addr"

lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Mnemonics.hs
199:        hrp = Bech32.unsafeHumanReadablePartFromText "addr"

lib/jormungandr/test/integration/Cardano/Faucet.hs
105:        let hrp = Bech32.unsafeHumanReadablePartFromText "addr"

lib/cli/src/Cardano/CLI.hs
355:        let hrp = Bech32.unsafeHumanReadablePartFromText "ed25519e_sk"

lib/core/src/Cardano/Wallet/Api/Types.hs
134:    ( dataPartFromBytes, dataPartToBytes, unsafeHumanReadablePartFromText )
1068:      where hrp = unsafeHumanReadablePartFromText "addr"

lib/core/src/Cardano/Wallet/Primitive/Types.hs
544:poolOwnerPrefix = Bech32.unsafeHumanReadablePartFromText "ed25519_pk"
```